src/play_game_logic.ts

```diff
@@ -1,15 +1,10 @@
-import { advanceTurn, createGameState, playerAssignWorkers, playerInitiateProject, TileUnderConstruction } from './game_logic'
-import { map } from './data/map'
-import { catalog as tileCatalog } from './data/tile-catalog'
-import { catalog as projectCatalog } from './data/project-catalog'
-import { hydrate } from './data/hydrate'
+import { projectCatalog } from './data/project-catalog'
+import { advanceTurn, createGameState, isTileUnderConstruction, playerAssignWorkers, playerInitiateProject } from './game_logic'
```

Comment:
I think this file is for demo/test purposes, so if it becomes costly to maintain I say we remove its

Reply:
It was a little helpful in designing things, but yeah.